clang-tools-extra/unittests/clang-doc/HTMLGeneratorTest.cpp

```diff
@@ -197,7 +197,9 @@ TEST(HTMLGeneratorTest, emitRecordHTML) {
     </p>
     <h2 id="Members">Members</h2>
     <ul>
-      <li>private int X</li>
+      <li>
+        <div>private int X</div>
```

Comment:
Should these be `private int X`, like this or should we have separate lists for `public`, `protected` and `private`?

Reply:
Either way this should be done in a separate change.

Reply:
Agreed.

Reply:
currently there's no distinction between public protected and private that's something i plan to change but not in this pr